api/mesh/v1alpha1/dataplane_helpers_test.go

```diff
@@ -356,6 +439,25 @@ var _ = Describe("Dataplane_Networking", func() {
 						{DataplaneIP: "192.168.0.1", DataplanePort: 443, WorkloadPort: 8443},
 					},
 				}),
+				Entry("2 inbound interfaces", testCase{
+					input: &Dataplane_Networking{
+						Address: "192.168.0.1",
+						Inbound: []*Dataplane_Networking_Inbound{
+							{
+								Port: 80,
+							},
+							{
+								Address:     "192.168.0.1",
```

Comment:
It's not clear whether `networking.inbound[].address` overrides `networking.address` or not